README.md

```diff
@@ -29,11 +29,11 @@ This repository contains all of the code that runs on [worldcubeassociation.org]
 
 ## Run directly with Ruby (lightweight, but only runs the Rails portions of the site)
 
-- Install either MySQL 5.6 or 5.7, and set it up with a user with username "root" with an empty password.
+- Install MySQL 8.0, and set it up with a user with username "root" with an empty password.
   If it poses problems, try the following:
   ```shell
-  # Install MySQL 5.6/5.7 if you haven't already.
-  # Using apt that would be: sudo apt install mysql-server-5.6
+  # Install MySQL 8.0 if you haven't already.
+  # Using apt that would be: sudo apt install mysql-server
```

Comment:
Does `mysql-server-8.0` work here instead?

Reply:
I thought of doing that at first, but it doesn't look like there's a package named like that?

https://packages.ubuntu.com/search?keywords=mysql-server&searchon=names&suite=disco&section=all